package.json

```diff
@@ -155,6 +155,7 @@
     "@babel/plugin-transform-object-assign": "^7.2.0",
     "@babel/preset-env": "^7.5.0",
     "@babel/preset-react": "^7.0.0",
+    "@cerner/eslint-config-terra": "^4.0.0",
```

Comment:
We need to update the config on line52 t pull from the new module:

Reply:
Updated in commit [f24ef00](https://github.com/cerner/terra-dev-site/pull/274/commits/f24ef003820f8b287bbecc331fb20f120c762591).